examples/experimental/bert_squad_pytorch/model_def.py

```diff
@@ -0,0 +1,166 @@
+from typing import Dict, Sequence, Union
+import torch
+import os
+import torch.nn as nn
+
+import determined as det
+from determined.pytorch import DataLoader, PyTorchTrial, LRScheduler
+import data
+import constants
+
+from transformers import (
+    AdamW,
+    get_linear_schedule_with_warmup,
+)
+from transformers.data.processors.squad import SquadResult
+from transformers.data.metrics.squad_metrics import (
+    compute_predictions_logits,
+    squad_evaluate,
+)
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+
+class BertSQuADPyTorch(PyTorchTrial):
+    def __init__(self, context: det.TrialContext) -> None:
+        self.context = context
+        self.download_directory = f"/tmp/data-rank{self.context.distributed.get_rank()}"
+        self.config_class, self.tokenizer_class, self.model_class = constants.MODEL_CLASSES[
+            self.context.get_hparam("model_type")
+        ]
+        self.validation_dataset, self.examples, self.features = data.load_and_cache_examples(
```

Comment:
non-blocking: `self.examples` -> `self.validation_examples` and `self.features` -> `self.validation_features` -- don't want somebody mixing these attributes up for the training examples / features.